cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/NodeVisitorAdapter.java

```diff
@@ -20,6 +20,11 @@ public void accept(ExpressionNode node) {
     }
 
     @Override
+    public void accept(ConstantNode node) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)